sdcflows/fieldmaps.py

```diff
@@ -251,12 +277,19 @@ def __attrs_post_init__(self):
 
             from .utils.epimanip import get_trt
 
+            msg = "Missing readout timing information for <%s>. %s"
+            extra = "Explicit fallback must be provided."
+            have_trt = False
             try:
                 get_trt(self.metadata, in_file=self.path)
-            except ValueError as exc:
-                raise MetadataError(
-                    f"Missing readout timing information for <{self.path}>."
-                ) from exc
+                have_trt = True
+            except ValueError:
+                with suppress(ValueError):
+                    get_trt(self.metadata, in_file=self.path, use_estimate=True)
+                    extra = "Estimated timing is available."
+
+            if not have_trt:
+                logger.warning(msg, self.path, extra)
```

Comment:
@oesteban Your review of changes to this class would be appreciated. I didn't see an obvious way to thread config options into `__attrs_post_init__`, so instead I settled for emitting warnings at object creation time. This may push errors down into runtime that would currently be caught. I'll see about having fMRIPrep run the check at workflow build time.

Reply:
Actually, best place to put it was in the sdcflows preproc workflow generator.